src/types/sequencing.ts

```diff
@@ -38,6 +44,17 @@ export type Parcel = {
   updated_at: string;
 };
 
+export type ParcelBundle = {
```

Comment:
What's the difference between a `Parcel` and a `ParcelBundle`? I think the "Parcel" name already implies that it's a bundle of objects.

Reply:
I wanted an object that merges all the data into one object-store. In the UI within the phoenix editor, I remember seeing code to get the selected parcel from that, we will use the parcel ID search the`parcelToParameterDictionaries` for the parcels. I want to replace that with the ParcelBundle has everything you need already stored. 

Let me know if that makes sense I can chat with you on slack about it.